### PR TITLE
[TIMOB-19215] Fix click events

### PR DESCRIPTION
**Pull request by hansemannn**
**(In progress)**

JIRA: https://jira.appcelerator.org/browse/TIMOB-19215

### CODE REVIEW

**Comment by kosso** [comment]
How's this going? I tried this PR version and still go no click events. 
A lot of people are stuck waiting for the annotation click events to return. 

thx.

**Comment by hansemannn** [comment]
@kosso : The PR ist in progress, we will merge and release a new version asap.

**Comment by cheekiatng** [comment]
APPROVED. merging.

File: apidoc/View.yml [hunk 1]
```diff
@@ -328,13 +328,9 @@ events:
 
         Note that only one annotation can be selected at any given time.
 
-        The `click` event includes a value, `clicksource`, which describes the part of the
-        annotation that was clicked. The `clicksource` can be one of `pin`, `title`,
-        `subtitle`, `leftPane`, `rightPane`, `infoWindow` or `null`. If the user clicks anywhere in
-        the info window other than `title`, `subtitle`, `leftPane` or `rightPane`, the
-        `clicksource` will be `infoWindow`. If the user deselects an annotation
+        The `click` event includes a value, `clicksource`, which describes the part of the annotation that was clicked. The `clicksource` can be one of `pin`, `leftButton` or `rightButton` on iOS and pin`, `title`, `subtitle`, `leftPane`, `rightPane`, `infoWindow` or `null` on Android. If the user deselects an annotation
```
Comment (ashcoding) [inline apidoc/View.yml]: Typo here: 
```javascript
on iOS and pin`, 
```
Requires a starting back-tick at the pin.

**Pull request merged**